concourse/pipelines/deployment-kick-off.yml

```diff
@@ -0,0 +1,59 @@
+---
+resource_types:
+- name: semver-iam
+  type: docker-image
+  source:
+    repository: governmentpaas/semver-resource
+    tag: dc33fee5c9061263e83159a893340f46728d93b5
+
+resources:
+  - name: paas-cf
+    type: git
+    source:
+      uri: https://github.com/alphagov/paas-cf.git
+      branch: {{branch_name}}
+      tag_filter: {{paas_cf_tag_filter}}
+      commit_verification_key_ids: {{gpg_ids}}
+
+  - name: deployment-timer
+    type: time
+    source:
+      days: [Monday, Tuesday, Wednesday, Thursday, Friday]
+      location: Europe/London
+      start: 7:45 AM
+      stop: 8:15 AM
+
+jobs:
+  - name: kick-off
+    serial: true
+    plan:
+      - get: deployment-timer
+        trigger: {{morning_deployment_trigger}}
+      - get: paas-cf
+
+      - task: await-turn
+        file: paas-cf/concourse/tasks/splay.yml
+        inputs:
+          - name: deployment-timer
```

Comment:
Established, it is not used but there to satisfy the set of tests we're running.

If the lines above would be taken away, travis would say, the source is defined, however not used.